frontend/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,16 @@
+import { Typography, Box } from '@material-ui/core';
+import useStyles from './styles'
+
+
+const Footer = ({text=`Copyright \u00a9 TourDeFoos 2021`}) => {
+    const classes = useStyles();
```

Comment:
Wytłmaczysz co to robi? Bo nigdy nie używałem, ale gdzieś tam widziałem :) ?

Reply:
To jest taki hook, który pozwala nam w kodzie JS stworzyć klasę wraz ze stylami i załadować je do komponentu

![image](https://user-images.githubusercontent.com/57037612/112978767-04496300-9158-11eb-963a-9b1411ebde91.png)

![image](https://user-images.githubusercontent.com/57037612/112978803-0f03f800-9158-11eb-9d63-20fd72f57b2a.png)